pkg/users/client.go

```diff
@@ -142,3 +144,62 @@ func (c *Client) GetUser(ctx context.Context, opts GetUserOpts) (User, error) {
 
 	return body, err
 }
+
+type Session struct {
+	ID        string `json:"id"`
+	Token     string `json:"token"`
+	CreatedAt string `json:"created_at"`
+	UpdatedAt string `json:"Updated_at"`
```

Comment:
```suggestion
	UpdatedAt string `json:"Updated_at"`
```
It should be `ExpiresAt`